bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roopythonize.py

```diff
@@ -44,15 +44,39 @@ def _fitTo(self, *args, **kwargs):
         return self._OriginalFitTo(*nargs)
 
 
+def _plotOn(self, *args, **kwargs):
+    """
+    Docstring
```

Comment:
I know you copy-pasted this ```Docstring``` line from the existing `_fitTo` function, but as Axel said this is not very meaningful just like that with no information.

For the existing `fitTo` function, it was there because we wanted to see if the docstring mechanism works in principle, i.e. that it gets shown when you type `?RooAbsReal.plotOn` in an ipython shell. Well, that worked, but we still have not figured out if we need such a docstring and what it should contain.

The problem is that the docstring should in principle document the function in all detail, but a detailed documentation already exists from the C++ code via doxygen. However, both the duplication of documentation and leaving the docstring empty are not good solutions for me.

So in conclusion, it's not clear yet what the docstring should contain yet, but I'll make sure we converge to a good solution before the next release. For this very welcome first PR of yours, I'm fine with this placeholder.

Reply:
Dear Mentor,

Sorry about this, when I was asked to change the documentation, I assumed it should be consistent with the one for _fitTo and since there was no input as such  about it from your end in the PR I had submitted as pre exercise, I assumed it is supposed to be that way. Is there anything I can do to improve this ?
Thanks